spec/osut_tests_spec.rb

```diff
@@ -2779,10 +2779,172 @@ module M
 
     # Independent point.
     p7 = OpenStudio::Point3d.new(14, 20, -5)
+    p8 = OpenStudio::Point3d.new(-9, -9, -5)
+
+    # Stress test 'to_p3Dv'. 4 valid input cases.
+    # Valid case #1: a single Point3d.
+    vtx = mod1.to_p3Dv(p0)
```

Comment:
All 4 valid `to_p3Dv` use cases systematically return an _OpenStudio::Point3dVector_ (and not occasionally an _Array_).